CONTRIBUTING.md

```diff
@@ -246,6 +246,17 @@ All donated code requires a license compatible with the Apache Software License
 2.0, and donated code will require a change of copyright to reflect the
 OpenTelemetry Authors.
 
+### Donation process
+
+Broadly, these are the steps the OpenTelemetry Governance and Technical
+Committees follow to handle a prospective donation.
+
+1. Per the above, the donating organization creates a GitHub issue (with the appropriate template) in the `community` repository.
```

Comment:
I assume such template will be provided soon? Currently there's no appropriate one in https://github.com/open-telemetry/community/tree/main/.github/ISSUE_TEMPLATE.

Reply:
yes, correct... can't add it to the PR, so kind of a race condition. once it's created I will update this to have a direct link.